stwcs/updatewcs/astrometry_utils.py

```diff
@@ -189,11 +189,13 @@ def updateObs(self, obsname):
                     headerlets[h].attach_to_file(obsname)
                 except ValueError:
                     pass
+        # Obtain the current primary WCS name
+        current_wcsname = obsname[('sci', 1)].header['wcsname']
         # Once all the new headerlet solutions have been added as new extensions
         # Apply the best solution, if one was specified, as primary WCS
         # This needs to be separate logic in order to work with images which have already
         # been updated with solutions from the database, and we are simply resetting.
-        if best_solution_id:
+        if best_solution_id and best_solution_id != current_wcsname:
```

Comment:
`best_solution_id` comes from the database entry. Is it populated/equivalent to `wcsname`?

It is the result of `getObservation` but is not listed in the `Returns` docstring. Can you add it there?

Reply:
These comments were addressed in 7260ca0.